### PR TITLE
feature/workday_scraper

### PR DESCRIPTION
**Pull request by noameron**
Added support for generic Workday scraper, and updated requirements.

I am facing issues trying to build docker, due to chrome / chromeDriver.

### CODE REVIEW

**Comment by nicobrenner** [comment]
> Added support for generic Workday scraper, and updated requirements.
> 
> I am facing issues trying to build docker, due to chrome / chromeDriver.

Saw your comment on the #23 ticket. Is that an error inside docker? Is your docker instance not able to install chrome? Are you able to run it using python + venv?

**Comment by nicobrenner** [comment]
Just looked over the PR, I like it. Please include a test for the scraper. It can be something very simple and straightforward that tests if the scraper is properly retrieving info from the site. Thank you

**Comment by noameron** [comment]
@nicobrenner Hey! I am able to run it locally through venv, and I also managed to build with docker just now but running it is causing some issues atm.

I pushed a commit with a successful docker build

File: docker/Dockerfile [hunk 1]
```diff
@@ -10,6 +10,11 @@ WORKDIR /commandjobs
 # Install any needed packages specified in requirements.txt
 RUN pip3 install --no-cache-dir -r config/requirements.txt
 
+# Install required packages, including Chromium and ChromeDriver
```
Comment (nicobrenner) [inline docker/Dockerfile:13]: Excellent :)

File: job_scraper/workday/scraper.py [hunk 2]
```diff
@@ -14,7 +16,7 @@
 class WorkdayScraper:
     def __init__(self, db_path='job_listings.db', update_func=None, done_event=None, result_queue=None):
         self.db_path = db_path
-        self.driver = webdriver.Chrome(options=self.get_selenium_configs())
+        self.driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()), options=self.get_selenium_configs())
```
Comment (nicobrenner) [inline job_scraper/workday/scraper.py:19]: Very helpful 👍🏼

File: src/menu.py [hunk 3]
```diff
@@ -66,8 +66,10 @@ def __init__(self, stdscr, logger):
         if self.total_ai_job_recommendations > 0:
             ai_recommendations_menu = f"✅ {self.total_ai_job_recommendations} recommended listings, out of {total_processed}"
 
-        self.menu_items = [resume_menu, "🕸  Scrape \"Ask HN: Who's hiring?\"",
+        self.menu_items = [resume_menu,
+                           "🕸  Scrape \"Ask HN: Who's hiring?\"",
                            "🕸  Scrape \"Work at a Startup jobs\"",
+                            "🕸  Scrape \"Workday\"",
```
Comment (nicobrenner) [inline src/menu.py:72]: 😄

**Comment by noameron** [comment]
@nicobrenner Hey! I just pushed a unit test for verifying the base elements of all job listings, I think next PR I should add a workflow for triggering the unit tests on each PR push.

Other than that everything seems to work on my end when i checked

**Comment by noameron** [comment]
Where should I update on the README regarding the new scarper for workday?

**Comment by nicobrenner** [comment]
> Where should I update on the README regarding the new scarper for workday?

Please add an entry at the top of the `Updates` > `Building in public:` section. It should be 2-3 sentences about what the change does in very simple terms so that most people reading the README can easily understand it

Optional: it would be great to have a short screencast of you demoing the new changes within `commandjobs`, you could add it with the above entry

Thank you

**Comment by noameron** [comment]
Just pushed the readme update! @nicobrenner

**Pull request merged**